Integrations/integration-jamf.yml

```diff
@@ -30,213 +30,164 @@ configuration:
-        var transformSingleObj= function(obj) {
-            var res = {};
-            mapFields.forEach(function(f) {
-                result = dq(obj, f.from);
-                console.log(result)
-                if (result || result === false) {
-                    res[f.to] = result;
-                }
-            });
-            if (filter && !filter(res)) {
-                return undefined;
+    base_url = demisto.params()['server']
+    user = demisto.params()['credentials']['identifier']
+    password = demisto.params()['credentials']['password']
+    auth = HTTPBasicAuth(user,password)
+
+    if not demisto.params()['proxy']:
+        del os.environ['HTTP_PROXY']
+        del os.environ['HTTPS_PROXY']
+        del os.environ['http_proxy']
+        del os.environ['https_proxy']
+
+    def jamf_get_computers():
+        endpoint = "/JSSResource/computers"
+        try:
+            contents = requests.get(base_url + endpoint,auth=auth,headers=HEADERS).json()
+            ec = {}
+            ec["JAMF"] = [{"ID": item["id"], "Name":item["name"]} for item in contents['computers']]
+
+            demisto.results({ 'Type' : entryTypes['note'], 'ContentsFormat' : formats['json'], 'Contents' : contents, "EntryContext": ec,"HumanReadable": tableToMarkdown("Computers", ec["JAMF"])})
+        except Exception,e:
+            demisto.results({ 'Type' : entryTypes['error'], 'ContentsFormat' : formats['text'], 'Contents' : "Unable to complete command.  Please check credentials and/or proxy settings" })
+            sys.exit(0)
+
+    def jamf_get_computers_match(match):
+        endpoint = "/JSSResource/computers/match/{match}"
+        try:
+            contents = requests.get(base_url + endpoint.format(match=match),auth=auth,headers=HEADERS).json()
+            ec = {}
+            ec["JAMF"] = [
+                {"ID": item["id"],
+                "Name": item["name"],
+                "AltMAC": item["alt_mac_address"],
+                "MAC": item["mac_address"],
+                "Serial": item["serial_number"],
+                "UDID": item["udid"],
+                "AssetTag": item["asset_tag"],
+                "BarCode-1": item["bar_code_1"],
+                "BarCode-2": item["bar_code_2"],
+                "Building": item["building"],
+                "BuildingName": item["building_name"],
+                "Department": item["department"],
+                "DepartmentName": item["department_name"],
+                "Email": item["email"],
+                "EmailAddress": item["email_address"],
+                "Position": item["position"],
+                "Username": item["username"],
+                "Realname": item["realname"],
+                "Room": item["room"]}
+                for item in contents["computers"]
+            ]
+
+            demisto.results({ 'Type' : entryTypes['note'], 'ContentsFormat' : formats['json'], 'Contents' : contents, "EntryContext": ec,"HumanReadable": tableToMarkdown("Computers", ec["JAMF"])})
```

Comment:
in case the `computers=[]` tableToMarkdown throws exception, need to check this case too.

Reply:
Also change in all places from one line JSON/dict to 
```
demisto.results({ 
                'Type' : entryTypes['note'], 
                'ContentsFormat' : formats['json'], 
                'Contents' : contents, 
                "EntryContext": ec,
                "HumanReadable": tableToMarkdown("Computers", ec["JAMF"])
})
```

Reply:
Be consistent, either use `'` single quote, or double `"`